integreat_cms/cms/templates/content_form_language_switcher.html

```diff
@@ -4,12 +4,12 @@
 {% load i18n %}
 
 <div id="language-switcher" class="flex flex-col z-10 text-lg">
-    <span id="language-switcher-current" href="{% build_url target=target region_slug=request.region.slug language_slug=language.slug content_field=content_field content_id=content_id %}" class="font-bold flex justify-between bg-white py-2 px-4 rounded shadow hover:bg-gray-200 border-2 border-white hover:border-gray-200">
-        Weitere Sprachen
+    <span id="language-switcher-current" class="font-bold flex justify-between bg-white py-2 px-4 rounded shadow hover:bg-gray-200 border-2 border-white hover:border-gray-200">
```

Comment:
```suggestion
    <span id="language-switcher-current" class="font-bold flex justify-between bg-white py-2 px-4 rounded shadow border-2 border-white">
```
I'm not sure about this, but if this area itself isn't a link anymore and clicking on it does nothing, maybe we should remove the hover effects and the pointer? But I don't know what's best practice for UI elements which trigger an action on _hovering_, not on _clicking_...